Add Intro & Getting Started sections to Eloqua docs (#4381)
segmentio/segment-docs: src/connections/destinations/catalog/eloqua/index.md

title: Eloqua Destination
id: 54521fd525e721e32a72eeac
---
Eloqua is Oracle’s B2B Cross-Channel Marketing solution, and it operates by tracking the identities and event actions of users that interact with your properties.

> With Segment, all the customer data you’re collecting from your websites, mobile apps, or servers gets sent directly into Eloqua, utilizing your existing Segment implementation. That means you don’t need to write any custom code or create duplicative event tracking! The result is that you can start using Eloqua with vastly simplified setup effort. 

Reviewer: ```suggestion
With Segment, all the customer data you’re collecting from your websites, mobile apps, or servers sends directly into Eloqua, using your existing Segment implementation. This means you don’t need to write any custom code or create duplicate event tracking, but you can start using Eloqua with a simplified setup effort. 
```

## Getting Started

> 1. From the Segment web app, navigate to **Connections > Catalog**.

Reviewer: ```suggestion
1. From the Segment web app, navigate to **Connections > Catalog** and select the **Destinations** tab within the catalog.
```

> 2. Search for **Eloqua** in the Destinations Catalog, and select the destination.

Reviewer: ```suggestion
2. Search for **Eloqua** and select the destination.
```

3. Click **Configure Eloqua**.
4. Select the web source that will send data to Eloqua and follow the steps to name your destination. 
5. On the **Settings** tab, input your **Company Name**, **Password**, **Site ID**, **Username**, and configure **Other Settings**.
6. Enable the destination.

> When you enable Eloqua in your Segment integrations page, Segment will automatically load Eloqua’s JavaScript libraries onto your web pages. For mobile and server-side events, Segment will route your event data directly through our servers, eliminating the need for any Eloqua code to be loaded in your applications.

Reviewer: ```suggestion
When you enable Eloqua in your Segment integrations page, Segment automatically loads Eloqua’s JavaScript libraries onto your web pages. For mobile and server-side events, Segment routes your event data directly through Eloqua's servers, eliminating the need for any Eloqua code to be loaded in your applications.
```
